src/components/ChatBotMessages/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import ChatBot from "react-simple-chatbot";
+import { ThemeProvider } from "styled-components";
+import { useTranslation } from "react-i18next";
+import womanBot from "./images/woman.svg";
+import womanSeeksHelp from "./images/womanSeeksHelp.svg";
+import "./index.scss";
+import { Redirect } from "react-router-dom";
+
+const CHATBOT_THEME = {
+  background: "#FFFEFC",
+  fontFamily: "Roboto",
+  headerBgColor: "#FFBFB5",
+  headerFontColor: "#fff",
+  headerFontSize: "15px",
+  botBubbleColor: "#C8D7C2",
+  botFontColor: "#fff",
+  userBubbleColor: "#FFBFB5",
+  userFontColor: "#fff",
+};
+
+const GoToSearchHelpPage = () => {
```

Comment:
collapse these three components into one component

notice how if you wanted to change the structure, say you wanted to use a div instead of a p, you have to change in 3 places

`const BotRedirectMessage = ({ path, message }) =>`

or name it whatever you think is suitable.

Reply:
Hope I did in the way you meant